evap/grades/forms.py

```diff
@@ -26,11 +25,17 @@ def __init__(self, *args, **kwargs):
             self.fields['last_modified_time_2'].widget = forms.HiddenInput()
             self.fields['last_modified_user_2'].widget = forms.HiddenInput()
 
-    def clean_description(self):
-        description = self.cleaned_data.get('description')
-        if GradeDocument.objects.filter(course=self.instance.course, description=description).exclude(id=self.instance.id).exists():
+    def clean_description_de(self):
+        description_de = self.cleaned_data.get('description_de')
+        if GradeDocument.objects.filter(course=self.instance.course, description_de=description_de).exclude(id=self.instance.id).exists():
             raise ValidationError(_("This description for a grade document was already used for this course."))
-        return description
+        return description_de
+
+    def clean_description_en(self):
+        description_en = self.cleaned_data.get('description_en')
+        if GradeDocument.objects.filter(course=self.instance.course, description_en=description_en).exclude(id=self.instance.id).exists():
```

Comment:
while you're at it, could you make this filter ignore case so we can catch more user errors here? ctrl-f for `iexact` in the django queryset docs.